docs/docs/spark-writes.md

```diff
@@ -201,6 +201,10 @@ Note WAP branch and branch identifier cannot both be specified.
 Also, the branch must exist before performing the write. 
 The operation does **not** create the branch if it does not exist. 
 For more information on branches please refer to [branches](branching.md).
+
+!!! info
```

Comment:
I think we also need some clarifications at https://iceberg.apache.org/docs/nightly/branching/